scripts/west_commands/runners/nrf_common.py

```diff
@@ -24,11 +24,26 @@
 ErrNotAvailableBecauseProtection = 24
```

Comment:
nit: drop `Flashing other cores like PPR/VPR is not supported yet` from f4be9e7ec2c05a2520547fcce65b2d70177fc004 message as it just works